openshift/minishift-addons/rhche/templates/rh-che.app.yaml

```diff
@@ -202,11 +202,6 @@ objects:
             value: ${CHE_FABRIC8_AUTH_ENDPOINT}
           - name: CHE_FABRIC8_MULTICLUSTER_OSO_PROXY_URL
             value: ${CHE_FABRIC8_MULTICLUSTER_OSO_PROXY_URL}
-          - name: CHE_DOCKER_ENABLE__CONTAINER__STOP__DETECTOR
```

Comment:
Are you sure that setting `CHE_DOCKER_ENABLE__CONTAINER__STOP__DETECTOR` to `false` isn't necessary anymore ? At some stage it was necessary to avoid some errors.

Reply:
It is not used in upstream Che at all